suite-native/discovery/src/discoveryConfigSlice.ts

```diff
@@ -60,13 +67,17 @@ export const discoveryConfigSlice = createSlice({
                 state.enabledDiscoveryNetworkSymbols.push(networkSymbol);
             }
         },
+        // setting all enabled coins at once
+        setEnabledDiscoveryNetworkSymbols: (state, { payload }: PayloadAction<NetworkSymbol[]>) => {
```

Comment:
Is this dispatched anywhere? I can't see it.

Reply:
not yet, but it should happen in followup Coin Enabling tasks. See the next comment pls.

Reply:
In that case I would prefer to add it in the followup PR 🙏

Reply:
removed 8f5b887272